modules/core/src/main/java/org/apache/ignite/internal/processors/failure/FailureProcessor.java

```diff
@@ -39,6 +40,12 @@
     private static final boolean IGNITE_DUMP_THREADS_ON_FAILURE =
         IgniteSystemProperties.getBoolean(IgniteSystemProperties.IGNITE_DUMP_THREADS_ON_FAILURE, true);
 
+    /** Ingonred failure log message. */
+    public static final String IGNORED_FAILURE_LOG_MESSAGE = "Possible failure suppressed accordingly to a configured handler ";
```

Comment:
Use `MSG` instead of `MESSAGE` for both constants accordingly to abbreviation rules.